samples/philosophers/sample.yaml

```diff
@@ -30,6 +30,7 @@ tests:
   sample.kernel.philosopher.lifos:
     extra_args: "-DFORKS=LIFOS"
   sample.kernel.philosopher.preempt_only:
+    platform_exclude: pinnacle_100_dvk
```

Comment:
why are you explicitly excluding this platform for so many samples? will you please explain in your commit message?

Reply:
A reason for some of these is that I could not add a board kcofnig overlay and get it to work. I believe there is an issue.  I filed it here https://github.com/zephyrproject-rtos/zephyr/issues/27934.